src/gui/FileBrowser.cpp

```diff
@@ -328,68 +328,62 @@ void FileBrowser::addItems(const QString & path )
 	}
 
 	// try to add all directories from file system alphabetically into the tree
-	QDir cdir( path );
-	QStringList files = cdir.entryList( QDir::Dirs, QDir::Name );
-	files.sort(Qt::CaseInsensitive);
-	for( QStringList::const_iterator it = files.constBegin();
-						it != files.constEnd(); ++it )
+	QDir cdir(path);
+	if (!cdir.isReadable()) { return; }
+	cdir.setNameFilters(m_filter.split(' '));
+	QFileInfoList entries = cdir.entryInfoList(
+			QDir::AllDirs | QDir::Files | QDir::NoDotAndDotDot,
+			QDir::LocaleAware | QDir::DirsFirst | QDir::Name | QDir::IgnoreCase);
+	for (auto& entry : entries)
```

Comment:
I realise the other existing loop uses the same style, but it's good practice to use const references unless you intend to modify the target of the reference.